package/cfg/rke2-cis-1.5/master.yaml

```diff
@@ -0,0 +1,1081 @@
+        For example,
+        chmod 644 $etcdconf
+      scored: true
+
+    - id: 1.1.8
+      text: "Ensure that the etcd pod specification file ownership is set to root:root (Scored)"
+      audit: "/bin/sh -c 'if test -e $etcdconf; then stat -c %U:%G $etcdconf; fi'"
+      tests:
+        test_items:
+          - flag: "root:root"
+            compare:
+              op: eq
+              value: "root:root"
+            set: true
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root $etcdconf
+      scored: true
+
+    - id: 1.1.9
+      text: "Ensure that the Container Network Interface file permissions are set to 644 or more restrictive (Not Scored)"
+      audit: "stat -c %a <path/to/cni/files>"
+      type: "manual"
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chmod 644 <path/to/cni/files>
+      scored: false
+
+    - id: 1.1.10
+      text: "Ensure that the Container Network Interface file ownership is set to root:root (Not Scored)"
+      audit: "stat -c %U:%G <path/to/cni/files>"
+      type: "manual"
+      remediation: |
+        Run the below command (based on the file location on your system) on the master node.
+        For example,
+        chown root:root <path/to/cni/files>
+      scored: false
+
+    - id: 1.1.11
+      text: "Ensure that the etcd data directory permissions are set to 700 or more restrictive (Scored)"
+      audit: ps -ef | grep $etcdbin | grep -- --data-dir | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %a
+      tests:
+        test_items:
+          - flag: "700"
+            compare:
+              op: eq
+              value: "700"
+            set: true
+      remediation: |
+        On the etcd server node, get the etcd data directory, passed as an argument --data-dir,
+        from the below command:
+        ps -ef | grep etcd Run the below command (based on the etcd data directory found above). For example,
+        chmod 700 /var/lib/etcd
+      scored: true
+      type: skip
+
+    - id: 1.1.12
+      text: "Ensure that the etcd data directory ownership is set to etcd:etcd (Scored)"
+      audit: ps -ef | grep $etcdbin | grep -- --data-dir | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%' | xargs stat -c %U:%G
+      tests:
+        test_items:
+          - flag: "etcd:etcd"
+            set: true
+      remediation: |
+        On the etcd server node, get the etcd data directory, passed as an argument --data-dir,
+        from the below command:
+        ps -ef | grep etcd
+        Run the below command (based on the etcd data directory found above).
+        For example, chown etcd:etcd /var/lib/etcd
+      scored: true
+      type: skip
+
+    - id: 1.1.13
+      text: "Ensure that the admin.conf file permissions are set to 644 or more restrictive (Scored)"
```

Comment:
This represents the the admin's kubeconfig file. RKE2 does in fact put such a file on disk, but at a different location (`/etc/rancher/rke2/rke2.yaml`).

This audit should be updated to test that.